src/game_loop.c

```diff
@@ -46,6 +46,8 @@
 #include <cdogs/character_class.h>
 #include <cdogs/player_template.h>
 
+#define PLAYER_TEMPLATE_FILE "players.cnf"
```

Comment:
this is already defined in player_template.c, can this be moved to a common location (e.g. player_template.h) instead?

Reply:
done, thanks!